Formula/ethermint.rb

```diff
@@ -0,0 +1,35 @@
+class Ethermint < Formula
+  desc "Ethereum powered by Tendermint consensus"
+  homepage "https://github.com/tendermint/ethermint"
+  url "https://github.com/tendermint/ethermint/archive/v0.5.4.tar.gz"
+  sha256 "b571f316841513efa16d8c62c8eeddac3f947af591f964aa3e3b1bb55f10f732"
+
+  bottle do
+    cellar :any_skip_relocation
+  end
+
+  head do
+    url "https://github.com/tendermint/ethermint.git",
+      :branch => "develop"
+  end
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ethermintpath = buildpath/"src/github.com/tendermint/ethermint"
+    ethermintpath.install buildpath.children
+    cd ethermintpath do
+      system "make", "get_vendor_deps"
```

Comment:
pinging @ilovezfs, not sure if this is accepted:

```
get_vendor_deps:
	@hash glide 2>/dev/null || go get github.com/Masterminds/glide
	@rm -rf vendor/
	@echo "--> Running glide install"
	@glide install
	@# ethereum/node.go:53:23: cannot use ctx (type *"github.com/tendermint/ethermint/vendor/gopkg.in/urfave/cli.v1".Context) as type *"github.com/tendermint/ethermint/vendor/github.com/ethereum/go-ethereum/vendor/gopkg.in/urfave/cli.v1".Context in argument to utils.SetEthConfig
	@rm -rf vendor/github.com/ethereum/go-ethereum/vendor
```

Reply:
could you be more specific? what's wrong with it?

Reply:
should be fine since the `go get` half won't run since we have `depends_on "glide" => :build`. This is actually equivalent to

```
system "glide", "install"
```

Usually we also set
```
ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
```